clang/test/AST/Interp/complex.cpp

```diff
@@ -37,6 +37,18 @@ constexpr _Complex int I2 = {};
 static_assert(__real(I2) == 0, "");
 static_assert(__imag(I2) == 0, "");
 
+constexpr int ignoredCast() {
+  I2;
+  (int)I2;
+  /* (float)I2; FIXME*/
+  D1;
+  /* (int)D1; FIXME*/
```

Comment:
The commented-out tests need https://github.com/llvm/llvm-project/pull/77295 first.